src/features/embalm/stepContent/hooks/useGetBalance.ts

```diff
@@ -42,5 +43,43 @@ export function useGetBalance() {
     })();
   }, [bundlr, dispatch, getBalance]);
 
+  useEffect(() => {
+    const queryBalance = async () => {
+      try {
+        const newBalance = await getBalance();
+        console.log('querying balance');
+        // If our new balance is different than current, then funding is complete
+        // update the new balance and clear out our interval
+        if (newBalance !== balance) {
+          console.log('balance is being updated due to funding completion');
+          dispatch(setPendingBalance({
+            balanceBeforeFund: '',
+            txId: ''
+          }));
+          dispatch(setBalance(newBalance));
+        }
+      } catch (error) {
+        console.error(error);
+      }
+    };
```

Comment:
leaving in these console logs temporarily for debugging until we have high confidence here